irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionService.java

```diff
@@ -151,12 +151,11 @@ public EdcCreatePolicyDefinitionRequest createPolicyDefinition(final String poli
                                              .type(POLICY_TYPE)
                                              .build();
 
-        final OdrlContext odrlContext = OdrlContext.builder().odrl(NAMESPACE_ODRL).build();
+        final Context context = Context.builder().odrl(NAMESPACE_ODRL).build();
 
         return EdcCreatePolicyDefinitionRequest.builder()
                                                .policyDefinitionId(accessPolicyId)
-                                               .policy(edcPolicy)
-                                               .odrlContext(odrlContext)
+                                               .policy(edcPolicy).context(context)
```

Comment:
please add the "cx-policy" namespace

Reply:
vocab, edc and cx-policy namespace have been added in the lines above